ee/clickhouse/queries/test/test_event_query.py

```diff
@@ -216,6 +217,23 @@ def test_account_filters(self):
 
         self._run_query(filter)
 
+    def test_actions(self):
```

Comment:
Is there anything specific about actions we’re testing here?  Maybe a test name or comment that states that this is ensuring that actions can be filtered by person properties?